diagnostic.md

```diff
@@ -62,7 +62,7 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
+undefined. The return a - b; does not have a definintion. Currently the code should only return a + b if a is greater than 0 or if its less than -5
```

Comment:
The if statement says that it would return a+b if "a is greater than 0 OR b is greater than 0." a is equal to 0, and b is less than 0, which means that the if statement evaluates to false. That means the next line that gets run is 57, which will return the value of 'a - b'. In this case, that would be '0 - (-5)' which equals 5.